src/certificate.tf

```diff
@@ -0,0 +1,111 @@
+locals {
+  max_length        = 24
+  alphanumeric_name = substr(replace(var.md_metadata.name_prefix, "/[^a-z0-9]/", ""), 0, local.max_length)
+}
+
+# This is trying to generate the certificates needed for Azure VPN. Not quite working yet. Still trying to figure out how to create the certificate pair for public/private key. Also need to figure out a way to make the certificates downloadable through the artifact.
```

Comment:
https://github.com/massdriver-cloud/artifact-definitions/blob/main/definitions/artifacts/kubernetes-cluster.json#L7-L13

This is how we add a downloadable Kubeconfig for `kubernetes-cluster` artifact. The template is base64 encoded - you can decode to see the content.